src/main/java/seedu/notus/storage/StorageManager.java

```diff
@@ -280,4 +310,62 @@ public boolean noteExists(Note note, boolean isArchive) {
         }
         return true;
     }
+
+    /* Timetable Saving and Loading */
+
+    /**
+     * Saves all the Events in the Timetable to the storage file.
+     *
+     * @param timetable The Timetable containing all the events to be saved.
+     */
+    public void saveTimetable(Timetable timetable) throws IOException {
+        String path = FOLDER_DIR + TIMETABLE_FILE_PATH;
+
+        //clear file
+        FileWriter fwClear = new FileWriter(path, false);
+        fwClear.write("");
+        fwClear.close();
+
+        //rewrite information to the file
+        FileWriter fwAppend = new FileWriter(path, true);
+
+        ArrayList<Event> nonRecurringEvents = timetable.getAllNonRecurringEvents();
+        String eventDetails;
+
+        for (Event event: nonRecurringEvents) {
+            eventDetails = getEventDetailsSaveFormat(event);
+            fwAppend.write(eventDetails);
+        }
+
+        ArrayList<RecurringEvent> recurringEvents = timetable.getAllRecurringEventsArray();
+
+        for (RecurringEvent event: recurringEvents) {
+            eventDetails = getEventDetailsSaveFormat(event);
+            eventDetails += PrefixSyntax.PREFIX_DELIMITER + PrefixSyntax.PREFIX_RECURRING
+                        + " " + event.getRecurrenceType() + " "
+                        + PrefixSyntax.PREFIX_DELIMITER + PrefixSyntax.PREFIX_STOP_RECURRING
+                        + " " + event.getEndRecurrenceDate() + " "
+                        + LS;
+            fwAppend.write(eventDetails);
+        }
+        fwAppend.close();
+    }
+
+    private static String getEventDetailsSaveFormat(Event event) {
+        String eventDetails;
+        eventDetails = PrefixSyntax.PREFIX_DELIMITER + PrefixSyntax.PREFIX_TITLE + " " + event.getTitle() + " "
+                + PrefixSyntax.PREFIX_DELIMITER + PrefixSyntax.PREFIX_TIMING + " " + event.getDateTime() + " ";
+
+        ArrayList<String> reminderPeriods = event.getReminderPeriodsString();
+
+        for (String reminderPeriod : reminderPeriods) {
+            eventDetails += PrefixSyntax.PREFIX_DELIMITER + PrefixSyntax.PREFIX_REMIND + " " + reminderPeriod + " ";
+        }
```

Comment:
Actually for reminder, it accepts the inputs as /remind 1-day 2-day 3-day. Not just /remind 1-day /remind 2-day

Reply:
Ah alright ok will fix that thanks!

Reply:
```suggestion
        eventDetails += (reminderPeriods.size() > 1) ? PrefixSyntax.PREFIX_DELIMITER + PrefixSyntax.PREFIX_REMIND  : "";
        for (String reminderPeriod : reminderPeriods) {
            eventDetails +=  " " + reminderPeriod;
        }
```